util/gogc.go

```diff
@@ -18,19 +18,19 @@ import (
 	"os"
 	"runtime/debug"
 	"strconv"
-	"sync/atomic"
+	"time"
 
 	"github.com/pingcap/tidb/metrics"
+	atomicutil "go.uber.org/atomic"
 )
 
-var gogcValue int64
+var gogcValue atomicutil.Int64 = *atomicutil.NewInt64(100)
```

Comment:
meet some races?

Reply:
It is code style guide in progress. if it is used as an atomic value, it should use ```uber/atomic``` to avoid forgetting the atomic operation.

Reply:
remove ```atomicutil```